### DISCUSSION TITLE
ERROR    old code "None" can not be upgraded

### DISCUSSION THREAD

**Discussion by ghost** [dashingsoft/pyarmor#1210] [Q&A]
### Discussed in https://github.com/dashingsoft/pyarmor/discussions/1204

<div type='discussions-op-text'>

<sup>Originally posted by **jamsavio** May  1, 2023</sup>
When I try activate my basic license, the pyarmor show this error message :

PS C:\Users\Jam\Desktop> pyarmor reg -u -p "Fortuno Blaze" 'C:\Users\Jam\Desktop\pyarmor-regcode-4064.txt'
INFO     Python 3.11.2
INFO     Pyarmor 8.1.9 (trial), 000000, non-profits
INFO     Platform windows.x86_64

Pyarmor 8.0 changes EULA and uses new commands
It's different from previous Pyarmor totally
Please read this import notes first:
https://github.com/dashingsoft/pyarmor/issues/980
Do not upgrade to Pyarmor 8 if don't know what are changed

I have known the changes of Pyarmor 8? (yes/no/help) yes
INFO     prepare "C:\Users\Jam\Desktop\pyarmor-regcode-4064.txt"
INFO     query key file from server
ERROR    please check Pyarmor 8.0 EULA
ERROR    old code "None" can not be upgraded</div>

**Comment by jondy** [comment]
I wonder why do you use `-u` to activate basic license? It's only used to upgrade old license (Pyarmor 7.x)
